site/create-image.md

```diff
@@ -99,6 +100,19 @@ installation or domain creation steps, use the `final-build-commands` section so
 final stage of the image build.  Or, if the file needs to change the Oracle Home prior to domain creation, use 
 the `after-fmw-install` or `before-wdt-command` sections.
 
+#### Resource Template Files
+
+If provided, the file or files provided with `--resourceTemplates` will be overwritten. For known tokens, 
+the placeholders will be replaced with values according to the table below.  **Note:** Placeholders must follow
```

Comment:
according to the table below -> according to the following table

Reply:
Thank you, fixed.